tests/beta/pydantic/test_basic.py

```diff
@@ -0,0 +1,151 @@
+from typing import List, Optional
+
+import pytest
+
+import pydantic
+
+import strawberry
+from strawberry.types.types import TypeDefinition
+
+
+def test_basic_type():
+    class User(pydantic.BaseModel):
+        age: int
+        password: Optional[str]
+
+    @strawberry.beta.pydantic.type(User, fields=["age", "password"])
+    class UserType:
+        pass
+
+    definition: TypeDefinition = UserType._type_definition
+
+    assert definition.name == "UserType"
```

Comment:
If there's a way to print the SDL for a single type, i'd add an assertion for that too. Primarily because developers aren't 'really going to be manually interrogating their types for `name`, `type` and `is_optional` very often, they're most concerned with what the end schema looks like.

Reply:
There is a `print_type` function available from graphql-core: https://github.com/graphql-python/graphql-core/blob/7d826f0ec0a447cb869dfb891a755dbe8ea9b66f/tests/utilities/test_build_ast_schema.py#L922-L927

Reply:
I will leave this test as it's right now as it's how we test the fields definition structure, and instead test the printed schema in the schema directory tests

Reply:
yeah, I think we'll provide our own custom print type in future, especially if we add better support for federation (now we had to hack the printer for showing the directives)